config/locales/en.yml

```diff
@@ -982,6 +982,7 @@ en:
               blank: Choose an outcome date
             date:
               blank: Enter an outcome date
+              future: Enter an outcome date in the past
```

Comment:
The two other dates where we validate 'not_in_future' have error messages like:

"Enter a graduation year that is in the past, for example 2014"

Maybe we should be consistent here, something like:

"Enter an outcome date that is in the past, for example 31 7 2020"

cc @gregknight1

Reply:
I'm no so sure of example `date` as there is additional constrained with the course start date

Reply:
@AbigailMcP @defong We shouldn't use examples in the error messages when there's already an example on the page so the other ones may need looking at - one for my ideas board.

And for dates, we should just say "Outcome date must be in the past"

I'll create a ticket for me to review date input validation messages so we can get them all consistent.

Reply:
Updated to be way nicer...

![image](https://user-images.githubusercontent.com/470137/132209131-c0a53bcc-86a7-4a74-b0fc-d3fcd8bc4927.png)